app/src/main/java/com/example/bigowlapp/activity/LoginPageActivity.java

```diff
@@ -112,8 +110,22 @@ protected void onStart() {
         }
     }
 
+    public void checkIfCanFingerprintAuthenticate() {
+        BiometricManager biometricManager = BiometricManager.from(this);
+        Intent i;
+        if (biometricManager.canAuthenticate() != BiometricManager.BIOMETRIC_SUCCESS) {
+            Toast.makeText(LoginPageActivity.this, "you are logged in!", Toast.LENGTH_SHORT).show();
+            i = new Intent(LoginPageActivity.this, HomePageActivity.class);
```

Comment:
just a note

btw, when you make intents/toasts like this, you can just say `this` for the first "context" parameter. You don't have to specify is like `LoginPageActivity.this`. That is only necessary when dealing with Anonymous classes.

Reply:
ok, good remark. Thanks